Source/Scene/Cesium3DTileset.js

```diff
@@ -381,7 +406,9 @@ define([
         }
 
         if (root.isContentUnloaded()) {
-            if (outOfCore) {
+            if (root.hasTilesetContent) {
```

Comment:
Are you sure this shouldn't be:
```javascript
if (outOfCore) {
    if (root.hasTilesetContent) {
        selectTileWithTilesetContent(tiles3D, selectedTiles, root, fullyVisible, frameState, true);
    } else {
        requestContent(tiles3D, root);
    }
}
```
When `outOfCore` is false, the tileset is rendered for picking and we want to render the same exact commands as the previous pass without making/processing any requests.